docs/api/python/text/text.md

```diff
@@ -121,13 +121,13 @@ by specifying their indices (2 and 1) and the `glossary.idx_to_vec` in
 ```
 
 
-## Glossary
+## CompositeEmbedding
 
 The glossary provides indexing and embedding for text tokens in a glossary. For
 each indexed token in a glossary, an embedding vector will be associated with
 it. Such embedding vectors can be loaded from externally hosted or custom
 pre-trained token embedding files, such as via instances of
-[`TokenEmbedding`](#mxnet.contrib.text.embedding.TokenEmbedding). 
+[`_TokenEmbedding`](#mxnet.contrib.text.embedding.TokenEmbedding). 
```

Comment:
This needs some work. There shouldn't be any reference to a private class.

Reply:
Do you also need to update the reference  `mxnet.contrib.text.embedding.TokenEmbedding `?